### PR TITLE
Health Exhibition and Confress

### PR DESCRIPTION
**Pull request by Habtamu210**
A capstone project for health exhibition and congresss
https://www.loom.com/share/5432c6f64f44468f8b219ce477342ed2

### CODE REVIEW

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Habtamu210, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- Commit messages are meaningful 👏 👏
- GitHub flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️
- Please use the New 👉  [ReadMe Template](https://github.com/microverseinc/readme-template) Provided by Microverse. **Note: Your project was supposed to be marked invalid based on this mistake. But because today is Saturday, you don't have much time. Kindly change this ReadMe** 
- Please, record again the video. Make sure it contains a highlight about a piece of the code you find interesting or proud of making.
- The README must contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
- Kindly make use of GitHub Pages to deploy the web pages. Make sure the project is deployed properly, and all pages, links, and images work as expected. The concerned link to the deployed project must be present in the README.
- 
- **PAGE NAVIGATION**:
- Make your website possible to navigate between the pages, there's a link from `Home` page to `About` page and the opposite too
- 
- **DESKTOP SIZE**:
- The top short bar section with social media links, logout, and language links aligned on **both pages(Home and About)**
- The top bar logo to the left must be the same logo used for the footer in both (Home and About)
- Capture1
- 
Capture
-
- **HOME PAGE**
- It contains a middle section with a column that includes five(5) individual articles, and not four(4). Kindly fix your design because it is a requirement of the project
- Capture
- **NOTE: Provide or Make visible the files on this PR for the Reviewer to make better inline comments.**
- Here is the Link to the 👉  [Project Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
   <title>capstone-project</title>
+  <script src="https://kit.fontawesome.com/c488aa6c77.js" crossorigin="anonymous"></script>
   <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
 </head>
 <body id="about-part">
```
Comment (Munsa1) [inline about.html:12]: - [ ] Kindly add the correct padding and margin so as to have the bottom navigation look like the one in the design

Your design
image

Given design
image

- [ ] The orange button could use more `bolder` styling so as to match the given design as well as reducing the padding so as to reduce the size of the button
- [ ] There should also be a link taking the user back to the index page in the about page

File: about.html [hunk 2]
```diff
@@ -82,7 +83,7 @@ <h2 class="text text-center">Health Exhibition and Confress logo</h2>
             trade fair in Ethiopia</p>
         </div>
         <div class="logo-b">
```
Comment (Munsa1) [inline about.html:85]: - [ ] This section has a heading , a paragraph of text and two articles. The paragraph is missing from your design, kindly fix this issue

Your design
image

Given design
image

File: about.html [hunk 3]
```diff
@@ -82,7 +83,7 @@ <h2 class="text text-center">Health Exhibition and Confress logo</h2>
...
-          <img src="images/pharmac.png" alt="Health Exhibition and Congress">
+          <img src="images/health-logo.png" alt="Health Exhibition and Congress">
         </div>
         <div class="spacing-3"></div>
```
Comment (Munsa1) [inline about.html:88]: - [ ] The footer section is not aligned as is in the given design kindly fix this as well

Your design
image

Given design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Habtamu210, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._

## Due to insifficient files changed I will leave some comments here

- [ ] In line 70 the `p` tag is supposed to be a `hx` tag such as h2, h3, h4 e.t.c. for semantic purposes, Kindly change it. This applies to the `about` page as well
image

- [ ] The main program is made up of five articles, kindly wrap each of these articles in an `article` tag instead of the div tag for semantic purposes
image

- [ ] Kindly add another logo to your design so as to match the given design
image



- [ ] Kindly remove the old readme file so as to have a cleaner looking project
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Qoosim — APPROVED** [review]
#   *Status: Approved* :tada: :heavy_check_mark: 

Hi @Habtamu210, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark:
-    Awesome UI. :heavy_check_mark: 




Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**